_api-reference/cat/cat-allocation.md

```diff
@@ -15,20 +15,36 @@ has_children: false
 The CAT allocation operation lists the allocation of disk space for indexes and the number of shards on each node.
 
 
-## Path and HTTP methods
-
+<!-- spec_insert_start
+api: cat.allocation
+component: paths_and_http_methods
+-->
+## Paths and HTTP methods
 ```json
-GET _cat/allocation?v
-GET _cat/allocation/<node_name>
+GET /_cat/allocation
+GET /_cat/allocation/{node_id}
 ```
+<!-- spec_insert_end -->
+
 
-## Query parameters
 
+<!-- spec_insert_start
+api: cat.allocation
+component: query_parameters
+-->
+## Query parameters
 Parameter | Type | Description
 :--- | :--- | :---
```

Comment:
We also need to specify defaults for the query parameters that have them (e.g., `local`).